src/lmdb/value_stream.cpp

```diff
@@ -38,7 +38,7 @@ namespace lmdb
     {
         std::size_t count(MDB_cursor* cur)
```

Comment:
The return type should be updated to `mdb_size_t` or `std::uint64_t`. I think `mdb_size_t` makes most sense, as the header already has the LMDB includes. I'll have to look to call if any callers need to update ...